modules/gapi/include/opencv2/gapi/cpu/video.hpp

```diff
@@ -17,9 +17,39 @@ namespace cpu {
 GAPI_EXPORTS GKernelPackage kernels();
 
 } // namespace cpu
+
+struct BackgroundSubtractorParams
+{
+    int history;
+    double threshold;
+    bool detectShadows;
+
+    BackgroundSubtractorParams()
+    {
+        history = 500;
+        threshold = 16;
+        detectShadows = true;
+    }
```

Comment:
These assignments could be done when defining class members, I think.
Not sure if it is legal to remove the default constructor at all, or preserve the string
```
BackgroundSubtractorParams() = default;
```

Reply:
Reworked